### PR TITLE
129 convert indexjs to typescript

### LINKED ISSUE #129: Convert index.js to typescript

### PR DESCRIPTION
**Pull request by dpenciso**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
     <div id="02a6e2cc"></div>
     <div id="fcf74975"></div>
     <!-- <div class="trelliscope-not-spa" id="80222985" style="width:800px; height:500px;"></div> -->
-    <div id="80222985"></div>
+    <div  id="80222985"></div>
```
Comment (Incin) [inline public/index.html:43]: i dont think you want this space?

Comment (dpenciso) [reply public/index.html:43]: you are correct

File: src/types/types.d.ts [hunk 2]
```diff
@@ -319,3 +319,7 @@ interface RelDispPositions {
   row: number;
   top: number;
 }
+
+interface Window {
+  trelliscopeApp: (id: string, config: string, options: { logger?: boolean; mockData?: any }) => void;
```
Comment (Incin) [inline src/types/types.d.ts:336]: is there no type for mockData?

Comment (dpenciso) [reply src/types/types.d.ts:336]: there is :)

**Review by Incin — APPROVED** [review]

**Pull request merged**